functions.php

```diff
@@ -443,7 +443,14 @@ function primer_register_sidebars() {
 
 	foreach ( $sidebars as $id => $args ) {
 
-		register_sidebar( array_merge( array( 'id' => $id ), $args ) );
+		register_sidebar(
+			array_merge(
+				array(
+					'id' => $id,
+				),
+				$args
+			)
+		);
```

Comment:
@fjarrett No, you are correct. This is more difficult to read. During the changes in this PR the new version of the WordPress coding standards was released, and I cam across this change. As per your other comment (on the PR for [Issue #200](https://github.com/godaddy/wp-primer-theme/pull/200)) you are correct. I've gone ahead and reverted this change to remain in scope of the PR.